packages/emails/email-manager.ts

```diff
@@ -695,6 +695,55 @@ export const sendAddGuestsEmails = async (calEvent: CalendarEvent, newGuests: st
 
   await Promise.all(emailsToSend);
 };
+
+export const sendAddGuestsEmailsAndSMS = async (args: {
```

Comment:
email-manager.ts is already a huge file. I would create a new file `guests-email-manager.ts` or something and put the new `sendAddGuestsEmailsAndSMS` function and functions it needs if it needs there